modules/heat_conduction/test/src/kernels/ADThermalConductivityTest.C

```diff
@@ -0,0 +1,34 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "ADThermalConductivityTest.h"
+
+registerADMooseObject("HeatConductionApp", ADThermalConductivityTest);
+
+defineADValidParams(ADThermalConductivityTest,
+                    ADMaterial,
+                    params.addRequiredCoupledVar("temperature", "Coupled temperature  ");
+                    params.addRequiredCoupledVar("c", "The coupled variable"););
```

Comment:
Maybe a little more descriptive here; maybe something like "Coupled variable used to help verify automatic differentiation capability"?

Reply:
Good idea